testem.js

```diff
@@ -12,12 +12,20 @@ let config = {
       "protocol": "tap"
      }
   },
+  "browser_args": {
+    "Chrome": [
```

Comment:
We can modify this so that it only does headless in CI (and in dev it does a normal chrome tab).  See https://twitter.com/trentmwillis/status/878277832278802432.